src/main/kotlin/no/nav/tiltakspenger/saksbehandling/behandling/domene/Behandling.kt

```diff
@@ -70,6 +74,7 @@ data class Behandling(
     val valgteTiltaksdeltakelser: ValgteTiltaksdeltakelser?,
     val avbrutt: Avbrutt?,
     val antallDagerPerMeldeperiode: Int?,
+    val avslagsgrunner: Set<Avslagsgrunnlag>,
```

Comment:
```suggestion
    val avslagsgrunner: NonEmptySet<Avslagsgrunnlag>?,
```

Er dette tydeligere? Hvis den er null er det enten ikke et avslag, eller så er den ikke behandlet ferdig. Hvis den ikke er null, må den minst ha 1 element.

Reply:
Sikkert greit nok det